content/docs/tasks/security/vault-ca/index.md

```diff
@@ -69,53 +66,74 @@ env:
     {{< /text >}}
 
-1.  List Node Agent's pods:
+1.  Deploy the example backends:
 
     {{< text bash >}}
-    $ kubectl get pod -n istio-system -l app=nodeagent -o jsonpath={.items..metadata.name}
+    $ kubectl apply -f httpbin-injected.yaml
+    $ kubectl apply -f sleep-injected.yaml
     {{< /text >}}
 
-1.  View each Node Agent's logs. The Node Agent residing on
-the same node as the testing workload will contain Vault related logs.
+## Istio mutual TLS with Vault CA integration
+
+This section provides a demo of Istio mutual TLS with Vault CA integration. With the previous steps,
+mutual TLS is enabled for the Istio deployment and the testing workloads `httpbin` and `sleep` receive
+certificates from the testing Vault CA. When sending a curl request from the `sleep` workload
+to the `httpbin` workload, the request goes through a mutual TLS protected channel constructed from
+the certificates issued by the Vault CA.
+
+1.  Send a `curl` request from the `sleep` workload to the `httpbin` workload.
+The request should succeed with a 200 response code since it goes through a mutual TLS protected channel
+constructed from the certificates issued by the Vault CA.
 
     {{< text bash >}}
-    $ kubectl logs -n istio-system THE-POD-NAME-FROM-PREVIOUS-COMMAND
+    $ kubectl exec -it $(kubectl get pod -l app=sleep -o jsonpath='{.items[0].metadata.name}') -c sleep -- curl -s -o /dev/null -w "%{http_code}" httpbin:8000/headers
+    200
     {{< /text >}}
 
-1.  Because in this example, Vault is not configured to accept the Kubernetes JWT
-service account from the `httpbin` workload, you should see that Vault rejects the
-signing requests with the following logs:
-
-    {{< text plain >}}
-    2019-01-16T19:42:19.274291Z     info    SDS gRPC server start, listen "/var/run/sds/uds_path"
-    2019-01-16T19:42:22.015814Z     error   failed to login Vault: Error making API request.
-    URL: PUT https://35.233.249.249:8200/v1/auth/kubernetes/login
-    Code: 500. Errors:
-    * service account name not authorized
-    2019-01-16T19:42:22.016112Z     error   Failed to sign cert for "default": failed to login Vault at https://35.233.249.249:8200: Error making API request.
+1.  Send a `curl` request from the `sleep` Envoy sidecar to the `httpbin` workload.
+The request should fail because the `httpbin` requires a mutual TLS connection while the request
```

Comment:
Please move this text after the command. That way, you provide readers with a clear logical chain.

```suggestion
    The request fails because the request from the sidecar to the `httpbin` workload did not use mutual TLS.  
```

Reply:
Done.